src/rprblender/nodes/blender_nodes.py

```diff
@@ -666,19 +666,59 @@ def _export_image_node(self, wrap_mapping):
             log.warn(f"Unsupported image wrap type {self.node.extension}")
             rpr_image.set_wrap(pyrpr.IMAGE_WRAP_TYPE_REPEAT)
 
-        # TODO: Implement using node properties: interpolation, projection
         if self.node.interpolation != 'Linear':
             log.warn("Ignoring unsupported texture interpolation", self.node.interpolation, self.node, self.material)
-        if self.node.projection != 'FLAT':
-            log.warn("Ignoring unsupported texture projection", self.node.projection, self.node, self.material)
 
-        rpr_node = self.create_node(pyrpr.MATERIAL_NODE_IMAGE_TEXTURE, {
-            pyrpr.MATERIAL_INPUT_DATA: rpr_image
-        })
+        if self.node.projection == 'BOX':
+            p = self.get_input_link('Vector')
+            if not p:
+                # this is a bit undefined.
+                p = self.create_node(pyrpr.MATERIAL_NODE_INPUT_LOOKUP, {
+                    pyrpr.MATERIAL_INPUT_VALUE: pyrpr.MATERIAL_NODE_LOOKUP_P_LOCAL
+                })
+            p_xy = p.get_channel(1).combine4(p.get_channel(0), 0.0, 0.0)
+            p_yz = p.get_channel(1).combine4(p.get_channel(2), 0.0, 0.0)
+            p_xz = p.get_channel(0).combine4(p.get_channel(2), 0.0, 0.0)
+
+            # lookup texture three times for each uv combo
+            tex_xy = self.create_node(pyrpr.MATERIAL_NODE_IMAGE_TEXTURE, {
+                pyrpr.MATERIAL_INPUT_DATA: rpr_image,
+                pyrpr.MATERIAL_INPUT_UV: p_xy
+            })
+            tex_yz = self.create_node(pyrpr.MATERIAL_NODE_IMAGE_TEXTURE, {
+                pyrpr.MATERIAL_INPUT_DATA: rpr_image,
+                pyrpr.MATERIAL_INPUT_UV: p_yz
+            })
+            tex_xz = self.create_node(pyrpr.MATERIAL_NODE_IMAGE_TEXTURE, {
+                pyrpr.MATERIAL_INPUT_DATA: rpr_image,
+                pyrpr.MATERIAL_INPUT_UV: p_xz
+            })
 
-        vector = self.get_input_link('Vector')
-        if vector:
-            rpr_node.set_input(pyrpr.MATERIAL_INPUT_UV, vector)
+            # calculate blend factor
+            blend = 1.0 / self.node.projection_blend if not math.isclose(self.node.projection_blend, 0.0) else 999.9
```

Comment:
blend value is never used there, can be removed

Reply:
ah this is an error, should be multiplied by weights.